generator/C/include_v2.0/mavlink_types.h

```diff
@@ -161,23 +161,43 @@ typedef struct __mavlink_message_info {
 #define mavlink_ck_a(msg) *((msg)->len + (uint8_t *)_MAV_PAYLOAD_NON_CONST(msg))
 #define mavlink_ck_b(msg) *(((msg)->len+(uint16_t)1) + (uint8_t *)_MAV_PAYLOAD_NON_CONST(msg))
 
-typedef enum {
```

Comment:
this will break pretty much all users of the library, as most users do define MAVLINK_COMM_NUM_BUFFERS and also use MAVLINK_COMM_0.
The common pattern is (MAVLINK_COMM_0+i)